softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/DeviceEntity.java

```diff
@@ -0,0 +1,79 @@
+package com.softgroup.common.dao.api.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import java.io.Serializable;
+
+/**
+ * Created by polev on 14.04.2017.
+ */
+@Entity
+public class DeviceEntity extends BaseEntity implements Serializable {
+    @Column (name = "profile_ID")
+    private String profile_id;
```

Comment:
@MariiaPolova Read java code convention. Underscore names are not allowed in Java.